apps/console/src/features/applications/components/wizard/help/generic-minimal-wizard-form-help.tsx

```diff
@@ -104,12 +104,10 @@ export const GenericMinimalWizardFormHelp: FunctionComponent<GenericMinimalWizar
                         {
                             (template.authenticationProtocol === SupportedAuthProtocolTypes.OIDC) && (
                                 <>
-                                    <Heading as="h5">Redirect URLs</Heading>
+                                    <Heading as="h5">Allowed redirect URLs</Heading>
                                     <p>
-                                        After the authentication, we will only redirect to the above redirect URLs. You
-                                        can
-                                        also
-                                        specify more than one URL if needed.
+                                        After the authentication, we will only redirect to the above allowed redirect
+                                        URLs. You can also specify more than one URL if needed.
```

Comment:
```suggestion
                                        After user sign-in or sign-out we will only redirect to these URLs. You can specify 
                                        multiple valid URLs. Make sure to specify the protocol ( https:// ) otherwise the 
                                        redirect may fail in some cases.
```

Reply:
Committed the change. Thanks.